workshop.md

```diff
@@ -416,38 +486,33 @@ As the code runs, you should see the generated questions and answers in the outp
 
 ## Create the Flashcards QR Codes
 
-Now that we have the questions and answers, we can generate QR codes for each flashcard. The QR code will point to the source material in the Microsoft Learn GitHub repository. In this way we follow [Responsible AI practices](https://www.microsoft.com/en-us/ai/responsible-ai) by providing the source material for each flashcard, so not only the user can learn more about the topic but also know where the LLM model got the information from.
-
-Add the following code to your notebook in a new cell to have a folder to store the QR codes:
+Now that we have the questions and answers, we can generate QR codes for each flashcard. The QR code will point to the source material in the Microsoft Learn GitHub repository. In this way we follow [Responsible AI](https://learn.microsoft.com/legal/cognitive-services/openai/overview?WT.mc_id=javascript-76678-cxa) practices by providing the source material for each flashcard, so not only the user can learn more about the topic but also know where the LLM model got the information from.
 
-```python
-import os
-
-# Path to QR Codes
-QR_CODE_PATH = f"{LAKEHOUSE_FILE_PATH}/qrcodes"
-os.makedirs(QR_CODE_PATH, exist_ok=True)
-```
-
-## Load the generated QAs
+Using the `qrcode` library we installed at the beginning, we can generate the QR codes for each flashcard. The source URL for the QR code is the URL to the Microsoft Learn module unit.
 
-Then we import the generated QAs from the JSON file we created before:
+to install the qrcode library, run the following command:
 
 ```python
-import json
-
-with open(f"{LAKEHOUSE_FILE_PATH}/generated-QAs.json", "r") as fp:
-    QAS = json.load(fp)
-
-QAS
+!pip install qrcode
```

Comment:
Wasn't this done at the start of the module?

Reply:
fixed this by moving the code to the Install Dependencies section